lib/siwapp_web/live/taxes_component.ex

```diff
@@ -113,4 +115,17 @@ defmodule SiwappWeb.TaxesComponent do
       Map.get(item.data, :taxes)
     end
   end
+
+  @spec unify_items_structure(map | list) :: map
```

Comment:
- Are you sure `params` can be a list?
- `normalize_items_params` could be a better name.

Reply:
Yep. There is a sentry proving it.

Agreed to the name annotation